eng/tools/RepoTasks/CreateFrameworkListFile.cs

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace RepoTasks
+{
+    public class CreateFrameworkListFile : Task
```

Comment:
Is this analogous to https://github.com/dotnet/core-setup/blob/5cfc2cab25bfc431ef89572aa2705725a95aca32/tools-local/tasks/CreateFrameworkListFile.cs? Should we move this to arcade and reduce the duplication?

Reply:
https://github.com/dotnet/arcade/issues/2704